modules/acquia_cms_starter/composer.json

```diff
@@ -9,7 +9,8 @@
         "drupal/acquia_cms_event": "1.x-dev",
         "drupal/acquia_cms_page": "1.x-dev",
         "drupal/acquia_cms_search": "1.x-dev",
-        "drupal/acquia_cms_video": "1.x-dev"
+        "drupal/acquia_cms_video": "1.x-dev",
+        "drupal/default_content": "^2.0"
```

Comment:
@rajeshreeputra This is already done in the feature branch i.e feature/moving-dependencies.

Reply:
we can not depends on the feature branch for PR.